README_zh.md

```diff
@@ -274,6 +274,7 @@ worksheet.properties.defaultRowHeight = 15;
 | outlineLevelCol  | 0          | 工作表列大纲级别 |
 | outlineLevelRow  | 0          | 工作表行大纲级别 |
 | defaultRowHeight | 15         | 默认行高 |
+| defaultColWidth  | (optional) | TBD |
```

Comment:
```suggestion
| defaultColWidth  | (可选) | 默认列宽 |
```

Reply:
I have no permission to commit it, @kigh-ota  could You?

Reply:
@Alanscut 
Thank you for you transalation!

@Siemienik 
Of course. I commited the same change.
(Maybe I should have turned on the GitHub's "Allow edits from maintainers" feature.)